helpers/bajasHelper/bajasHelper.helper.go

```diff
@@ -414,22 +444,104 @@ func GetDetalleElemento(id int) (Elemento *models.DetalleElementoBaja, outputErr
 		ubicacion = ubicacion_
 	}
 
-	if funcionario_, err := tercerosMidHelper.GetFuncionario(int(detalleJSON["funcionario"].(float64))); err != nil {
+	if funcionario_, err := tercerosMidHelper.GetInfoTerceroById(int(detalleJSON["funcionario"].(float64))); err != nil {
 		return nil, err
 	} else {
 		funcionario = funcionario_
 	}
 
-	Elemento.Id = elementoMovimiento[0].Id
+	Elemento.Id = elementoMovimiento.Id
 	Elemento.Placa = elemento[0].Placa
 	Elemento.Nombre = elemento[0].Nombre
 	Elemento.Marca = elemento[0].Marca
 	Elemento.Serie = elemento[0].Serie
 	Elemento.SubgrupoCatalogoId = elemento[0].SubgrupoCatalogoId
-	Elemento.Salida = elementoMovimiento[0].MovimientoId
+	Elemento.Salida = elementoMovimiento.MovimientoId
 	Elemento.Ubicacion = ubicacion
-	Elemento.Funcionario = funcionario[0]
+	Elemento.Funcionario = funcionario
 
-	fmt.Println(elemento, elementoMovimiento)
 	return Elemento, nil
 }
+
+func GetDetalleElementos(ids []int) (Elementos []*models.DetalleElementoBaja, outputError map[string]interface{}) {
```

Comment:
Falta documentar, cambiar `...` en el _suggested change_, digamos "GetDetalleElementos hace tal cosa"
```suggestion
// GetDetalleElementos ...
func GetDetalleElementos(ids []int) (Elementos []*models.DetalleElementoBaja, outputError map[string]interface{}) {
```